pkgs/misc/vim-plugins/default.nix

```diff
@@ -1309,27 +1309,29 @@ rec {
     dependencies = [];
     buildInputs = [
       python go cmake
-      (if stdenv.isDarwin then llvmPackages.clang else llvmPackages.clang-unwrapped)
-      llvmPackages.llvm
     ] ++ stdenv.lib.optional stdenv.isDarwin Cocoa;
 
     propagatedBuildInputs = stdenv.lib.optional (!stdenv.isDarwin) rustracerd;
 
     patches = [
-      ./patches/youcompleteme/1-top-cmake.patch
       ./patches/youcompleteme/2-ycm-cmake.patch
     ];
 
+    # YCM requires path to external libclang 3.9
```

Comment:
But this is true
https://github.com/Valloric/YouCompleteMe/blob/master/README.md
```
YCM is designed to work with libclang version 3.9 or higher.
```